src/gpu/intel/ocl/micro_sdpa.cpp

```diff
@@ -264,7 +264,10 @@ status_t micro_sdpa_t::pd_t::init_microkernels(impl::engine_t *engine) {
 
     problem_kq.B.layout = MatrixLayout::Pr;
     problem_kq.C.layout = MatrixLayout::T;
-    problem_kq.A.setAlignment(alignmentForLD(d->head_size() * problem.Ta));
+    const memory_desc_wrapper key_mdw(key_md());
+    auto ldk = static_cast<int>(
+            gemm_desc_t::get_ld(*key_md()) * key_mdw.data_type_size());
+    problem_kq.A.setAlignment(alignmentForLD(ldk));
```

Comment:
Thanks for fixing this! Can you also fix the alignment for `problem_vs.A` on line 337? It should be based on the alignment of `ldv`, similar to the definition of `V_ALIGN` on line 415.